sources/packages/web/src/components/common/students/StudentProfile.vue

```diff
@@ -128,7 +128,8 @@ export default defineComponent({
   setup(props) {
     const studentDetail = ref({} as SharedStudentProfile);
     const address = ref({} as AddressAPIOutDTO);
-    const { genderDisplayFormat, sinDisplayFormat, emptyStringFiller } = useFormatters();
+    const { genderDisplayFormat, sinDisplayFormat, emptyStringFiller } =
```

Comment:
is this change intentional?, else please revert.

Reply:
It's an old formatter update that didn't get put into another PR.